modules/diskq/tests/test_diskq_truncate.c

```diff
@@ -257,6 +258,123 @@ Test(diskq_truncate, test_diskq_truncate_on_push)
   _save_diskqueue(q);
 }
 
+static void
+_assert_cursors_are_at_start(LogQueue *q)
+{
+  QDisk *qdisk = ((LogQueueDisk *)q)->qdisk;
+
+  cr_assert_eq(qdisk->hdr->read_head, QDISK_RESERVED_SPACE, "Read head was not reset!");
+  cr_assert_eq(qdisk->hdr->write_head, QDISK_RESERVED_SPACE, "Write head was not reset!");
+  cr_assert_eq(qdisk->hdr->backlog_head, QDISK_RESERVED_SPACE, "Backlog head was not reset!");
+}
+
+Test(diskq_truncate, test_diskq_truncate_size_ratio_default)
+{
+  const gint truncate_size_ratio = (gint)(1100000 * 0.01);
+  const gint below_threshold_message_number = truncate_size_ratio / 134;
```

Comment:
As far as I can see, these "magic numbers" are the consequences of `feed_some_messages` which is generating `log_msg_new_empty` messages. And our code is highly rely on the current size of those messages.

Thus, I am not expecting to refactor the whole unit test system to use "sizeof" functions. However I highly recommend to name every constant before it's use. Like you did with the `truncate_size_ratio`.

i.e.: `const gint log_msg_size_dependent_divider = 134;`

Reply:
Thanks, fixed.